sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/AbstractStorageTest.java

```diff
@@ -117,11 +101,23 @@
             "    \"_etag\": \"%s\",\n" +
             "    \"_attachments\": \"attachments/\",\n" +
             "    \"_ts\": 1550881731\n" +
-            "}", TEST_FIELD_VALUE, DOCUMENT_ID, PARTITION, ETAG);
+            "}", TEST_FIELD_VALUE, DOCUMENT_ID, RESOLVED_USER_PARTITION, ETAG);
+    static String mUserTableName = Utils.getUserTableName(AbstractStorageTest.ACCOUNT_ID);
```

Comment:
static is sField, static final is FIELD, either way is not mField.

Also blank lines between fields